spaceship/lib/spaceship/tunes/app_export_compliance.rb

```diff
@@ -0,0 +1,46 @@
+module Spaceship
+  module Tunes
+    # Represents the encryption export compliance information for an app
+    class AppExportCompliance < TunesBase
+      # @return
+      attr_reader :id
+      attr_reader :upload_date
+      attr_reader :code_value
+      attr_reader :platform
+      attr_reader :status
+      attr_reader :export_compliance_compliance_required
+      attr_reader :uses_non_exempt_encryption_from_plist
+      attr_reader :ccat_file
+      attr_reader :builds
+      attr_reader :store_versions
+
+      attr_mapping({
+        'id' => :id,
+        'uploadDate' => :upload_date,
+        'codeValue' => :code_value,
+        'platform' => :platform,
+        'status' => :status,
+        'exportComplianceRequired' => :export_compliance_compliance_required,
```

Comment:
@mfurtak:I have updated the PR, fixing the above.

I just realized that there was some implementation of export compliance under app_submission: 
https://github.com/fastlane/fastlane/blob/7b5e166/spaceship/lib/spaceship/tunes/app_submission.rb#L82-L92

I could try refactoring a bit, or do it later. WDYT ?